js/index.js

```diff
@@ -0,0 +1,98 @@
+import lozad from "lozad";
+import checkElement from "../utils/dom";
+
+import Menu from "./models/menu";
+import * as menuView from "./views/menuView";
+
+import * as navMenus from "../utils/navMenus";
+
+const state = {};
+
+function addDisplayOnHover() {
+  const elementContained = document.querySelectorAll(".article");
+  let clickedElement;
+  Array.from(elementContained).forEach(async elem => {
+    elem.addEventListener("mouseover", function() {
+      const id = elem.getAttribute("class").split(" ");
+      if (clickedElement) {
+        clickedElement.classList.remove("hidden");
+      }
+      const hoveredElement = document.querySelector(`div.${id[1]}`);
+      clickedElement = hoveredElement;
+      document.querySelector(`div.${id[1]}`).classList.toggle("hidden");
+    });
+  });
+}
+const controlMenu = async () => {
+  if (!state.menu) state.menu = new Menu();
+
+  const elemsData = await navMenus.elems;
+  const elems = await Promise.all(
```

Comment:
I think you should re-check the use of `async/await` on this line

Reply:
👍

Reply:
each element inside `elemsData` have a promise inside of its property and is resolve inside the loop inside each element in `elems` is something wrong with that. please let me know.